src/Symfony/Component/Form/Extension/Core/Type/CountryType.php

```diff
@@ -38,7 +38,13 @@ public function configureOptions(OptionsResolver $resolver)
     {
         $resolver->setDefaults(array(
             'choice_loader' => function (Options $options) {
-                return $options['choices'] ? null : $this;
+                if ($options['choices']) {
+                    @trigger_error(sprintf('Using "choices" option in %s has been deprecated since version 3.3 and will be ignored in 4.0. Override the "choice_loader" option instead.', __CLASS__), E_USER_DEPRECATED);
```

Comment:
I hesitated, considering your comments I'd rather add it explicitly here too, thanks.